--- mkimage/debootstrap	2015-12-30 09:41:25.800971000 +0000
+++ mkimage/debootstrap	2015-12-30 09:51:25.218188600 +0000
@@ -149,6 +149,18 @@
 		# https://aptitude.alioth.debian.org/doc/en/ch02s05s05.html#configApt-AutoRemove-SuggestsImportant
 		Apt::AutoRemove::SuggestsImportant "false";
 	EOF
+
+	# update "install suggests" configuration
+	echo >&2 "+ echo Apt::Install-Suggests 'false' > '$rootfsDir/etc/apt/apt.conf.d/docker-install-suggests'"
+	cat > "$rootfsDir/etc/apt/apt.conf.d/docker-install-suggests" <<-'EOF'
+		APT::Install-Suggests "false";
+	EOF
+
+	# update "install recommends" configuration
+	echo >&2 "+ echo Apt::Install-Recommends 'false' > '$rootfsDir/etc/apt/apt.conf.d/docker-install-recommends'"
+	cat > "$rootfsDir/etc/apt/apt.conf.d/docker-install-recommends" <<-'EOF'
+		APT::Install-Recommends "false";
+	EOF
 fi
 
 if [ -z "$DONT_TOUCH_SOURCES_LIST" ]; then
