diff --git rootfs/mkimage/debootstrap rootfs/mkimage/debootstrap
index 7046b6e..1193c4e 100755
--- rootfs/mkimage/debootstrap
+++ rootfs/mkimage/debootstrap
@@ -179,0 +180,12 @@ if [ -d "$rootfsDir/etc/apt/apt.conf.d" ]; then
+
+	# update "install suggests" configuration
+	echo >&2 "+ echo Apt::Install-Suggests 'false' > '$rootfsDir/etc/apt/apt.conf.d/docker-install-suggests'"
+	cat > "$rootfsDir/etc/apt/apt.conf.d/docker-install-suggests" <<-'EOF'
+		APT::Install-Suggests "false";
+	EOF
+
+	# update "install recommends" configuration
+	echo >&2 "+ echo Apt::Install-Recommends 'false' > '$rootfsDir/etc/apt/apt.conf.d/docker-install-recommends'"
+	cat > "$rootfsDir/etc/apt/apt.conf.d/docker-install-recommends" <<-'EOF'
+		APT::Install-Recommends "false";
+	EOF
@@ -181,0 +194,3 @@ fi
+# sources list is empty :/, adding mirror manually
+[[ -s "$rootfsDir/etc/apt/sources.list" ]] || echo "deb ${@: -1} $suite main" >> "$rootfsDir/etc/apt/sources.list"
+
