--- rootfs/mkimage/debootstrap
+++ rootfs/mkimage/debootstrap
@@ -188,6 +188,9 @@ if [ -d "$rootfsDir/etc/apt/apt.conf.d" ]; then
 	EOF
 fi
 
+# sources list is empty :/, adding mirror manually
+[[ -s "$rootfsDir/etc/apt/sources.list" ]] || echo "deb ${@: -1} $suite main" >> "$rootfsDir/etc/apt/sources.list"
+
 if [ -z "$DONT_TOUCH_SOURCES_LIST" ]; then
 	# tweak sources.list, where appropriate
 	lsbDist=
