--- rootfs/mkimage.sh	2016-01-07 19:30:07.591254300 +0000
+++ rootfs/mkimage.sh	2016-01-07 19:31:18.798092700 +0000
@@ -107,6 +107,21 @@
 
 if [ "$tag" ]; then
 	( set -x; docker build -t "$tag" "$dir" )
+cat > "$dir/Dockerfile" <<EOF
+FROM $tag
+ENV LANG C.UTF-8
+ENV DEBIAN_FRONTEND noninteractive
+RUN echo '\
+path-exclude /usr/share/doc/*\n\
+path-include /usr/share/doc/*/copyright\n\
+path-exclude /usr/share/man/*\n\
+path-exclude /usr/share/groff/*\n\
+path-exclude /usr/share/info/*\n\
+path-exclude /usr/share/lintian/*\n\
+path-exclude /usr/share/linda/*\n\
+' >> /etc/dpkg/dpkg.cfg.d/01_nodoc
+EOF
+	( set -x; docker build -t "$tag" "$dir" )
 elif [ "$delDir" ]; then
 	# if we didn't specify a tag and we're going to delete our dir, let's just build an untagged image so that we did _something_
 	( set -x; docker build "$dir" )
